server/server.go

```diff
@@ -114,6 +117,7 @@ func DefaultConfig() Config {
 		TLSCAFile:      "",
 		TLSCrtFile:     "",
 		TLSKeyFile:     "",
+		Viper:          viper.New(),
```

Comment:
Clients can now configure a viper in their server factory and use it for whatever they need it.